app/src/pages/inside/stepPage/stepGrid/defectType/defectType.jsx

```diff
@@ -92,18 +98,29 @@ export const DefectType = ({ issue, onEdit, onRemove, patternTemplates, events,
         {issue.issueType && (
           <DefectTypeItem
             type={issue.issueType}
-            onClick={disabled ? null : () => onClickEdit(events.onEditEvent?.(eventData))}
+            onClick={
+              disabled || canChangeDefectTypes
```

Comment:
Should it be reversed to
```suggestion
              disabled || !canChangeDefectTypes
```
?